service/src/main/java/uk/gov/hmcts/reform/fpl/handlers/GeneratedOrderEventHandler.java

```diff
@@ -39,26 +44,44 @@
     private final RepresentativeNotificationService representativeNotificationService;
     private final IssuedOrderAdminNotificationHandler issuedOrderAdminNotificationHandler;
     private final SendDocumentService sendDocumentService;
+    private final FeatureToggleService featureToggleService;
 
     @EventListener
     public void notifyParties(final GeneratedOrderEvent orderEvent) {
         final CaseData caseData = orderEvent.getCaseData();
         final DocumentReference orderDocument = orderEvent.getOrderDocument();
 
         issuedOrderAdminNotificationHandler.notifyAdmin(caseData, orderDocument, GENERATED_ORDER);
-        sendNotificationToLocalAuthorityAndDigitalRepresentatives(caseData, orderDocument);
 
-        sendNotificationToEmailServedRepresentatives(caseData, orderDocument);
+        if (featureToggleService.isSendOrderToOthersEnabled()) {
```

Comment:
something that can be checked wether is a new or old event rather? isNewEvent() on GeneratedOrder